src/lib/util/random-id.ts

```diff
@@ -1,5 +1,5 @@
-import { v4 as uuidv4 } from 'uuid';
+import { randomUUID } from 'node:crypto';
 
 export const randomId = (): string => {
-    return uuidv4();
+    return randomUUID();
```

Comment:
Does this abstraction add anything for us? Less refactor? Seems like an abstraction that adds very little

Reply:
It does very little... maybe the idea was to use randomId instead of the specific underlying function, but we're not doing that...